module/metrics/example/consensus/main.go

```diff
@@ -63,8 +64,8 @@ func main() {
 				collector.FinishBlockToSeal(flow.HashToID(entityID))
 			}
 
-			collProvider := engine.TestNetwork.String()
-			collIngest := engine.TestMetrics.String()
+			collProvider := network.TestNetwork.String()
+			collIngest := network.TestMetrics.String()
```

Comment:
No it causes weird import cycles because we would still need to import them into the network package and add them to the authorized roles mapping for channel participation. I renamed them for more readability.

Reply:
https://github.com/onflow/flow-go/pull/2491/commits/2d73cd4855599a1b12a5e65f1d615a50624021e5